packages/studio-web/src/app/shepherd.steps.ts

```diff
@@ -145,16 +145,170 @@ export const language_step: any = {
+  text: $localize`Clicking on an individual word will play it as well.`,
+  buttons: [
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Back`,
+      type: "back",
+    },
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Next`,
+      type: "next",
+    },
+  ],
+  id: "readalong-play-word",
+};
+
+export const readalong_change_title_step: any = {
+  title: $localize`Edit your ReadAlong\n`,
+  text: $localize`You can edit the text here to add a Title or Subtitle to your ReadAlong.`,
+  attachTo: {
+    element: "input",
+    on: "bottom"
+  },
+  buttons: [
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Back`,
+      type: "back",
+    },
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Next`,
+      type: "next",
+    },
+  ],
+  id: "readalong-change-title",
+};
+
+export const readalong_add_image_step: any = {
+  title: $localize`Edit your ReadAlong\n`,
+  text: $localize`You can add an image to each page of your ReadAlong here.`,
+  buttons: [
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Back`,
+      type: "back",
+    },
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Next`,
+      type: "next",
+    },
+  ],
+  id: "readalong-add-image",
+};
+
+export const readalong_add_translation_step: any = {
+  title: $localize`Edit your ReadAlong\n`,
+  text: $localize`You can add a translation or extra bit of information to each sentence of your ReadAlong by clicking on the + icon here. If you later want to remove the translation, click on the − icon after adding your translation.`,
+  buttons: [
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Back`,
+      type: "back",
+    },
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Next`,
+      type: "next",
+    },
+  ],
+  id: "readalong-add-translation",
+};
+
+export const readalong_export_step: any = {
+  title: $localize`Export your ReadAlong\n`,
+  text: $localize`When you're happy with your ReadAlong, you can export it by selecting a download format and then clicking on the download button (down arrow). You can choose from the following formats: `,
+  attachTo: {
+    element: "div.download__buttons",
+    on: "bottom"
+  },
+  buttons: [
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Back`,
+      type: "back",
+    },
+    {
+      classes: "shepherd-button-primary",
+      text: $localize`Next`,
+      type: "next",
+    },
+  ],
+  id: "readalong-export",
+};
+
+export const readalong_go_back_step: any = {
+  title: $localize`Edit your ReadAlong\n`,
+  text: $localize`If you'd like to change your audio or text you can always go back to Step 1 and change things or make a new ReadAlong, but be careful! Going back to the previous step will mean you will lose all of your translations and images.`,
+  attachTo: {
+    element: "div.mat-horizontal-stepper-header-container",
```

Comment:
Already mentioned in Slack, but it's worth noting here in the PR: this selector depends on Angular innards that could change in future versions, so it may not be stable. However, the element we're trying to attach to is a generated one, so I don't see any obvious fix.